garage/tf/samplers/off_policy_vectorized_sampler.py

```diff
@@ -29,16 +29,16 @@ def __init__(self, algo, n_envs=None):
         :param algo: Algorithms.
         :param n_envs: Number of parallelized sampling envs.
         """
-        super(OffPolicyVectorizedSampler, self).__init__(algo)
+        if n_envs is None:
+            n_envs = int(algo.rollout_batch_size)
+            n_envs = max(1, min(n_envs, 100))
```

Comment:
i get what you're doing here, but i think an `if` would be 100% clearer

also, why limit to 100? we have some big computers.

Reply:
Removed 100 limit.